### PR TITLE
Update gdsway index

### PR DESCRIPTION
**Pull request by soupdragon99**
Updated with ref to @philandstuff suggestions.

### CODE REVIEW

File: source/index.html.md.erb [hunk 1]
```diff
@@ -1,21 +1,33 @@
...
 
 # <%= current_page.data.title %>
 
+# What is The GDS Way?
+
+Since GDS was formed a few years ago, we've grown to an organisation of several hundred people with well established teams which are meeting user needs by creating software.
+
+As we've grown the teams have all had to choose how they want to build their services. In some ways this has been good - every team has picked the most suitable tools for their project - but now we suspect that we're at the point where having some easy-to-use defaults for new teams will be helpful. Some of the things we're hoping to achieve are:
+
+* using similar tools across the organisation will make people more productive when they move between teams, meaning we might be able to do that faster and more often (assuming people want to)
```
Comment (oliverpartridge) [inline source/index.html.md.erb]: Do we need to say "(assuming people want to)" ?

Comment (soupdragon99) [inline source/index.html.md.erb]: Thanks, I wanted to keep the content slightly informal in keeping with GDS culture.

**Comment by soupdragon99** [comment]
Added links to the service manual and the digital service standard as suggested.

**Comment by bruntonspall** [comment]
Looks good to me

**Comment by J-Lambo** [comment]
Content looks good. Some small suggestions to bring it in line with our style. I've discussed these in person with @soupdragon99, but in summary:

- Amend intro line to make shorter. 'The GDS Way is ... If you do not work at GDS, please refer to the Service Manual[link] for guidance on building government services. 

- Avoid using questions in sub-headings. Change to 'About GDS Way' or similar. 

- Condense list of bullet points to work with lead-in line. 

- Use 'includes' rather than 'encompasses'. 

- Beware switching between 'us' and 'we' and back to 'GDS'. Stick to one for consistency.

**Comment by soupdragon99** [comment]
Thanks @J-Lambo for the review, I have now made some style edits based on your feedback.

**Comment by philandstuff** [comment]
merging since it's been through a sort of 2i with @soupdragon99 and @J-Lambo

**Pull request merged**